README.md

```diff
@@ -25,13 +25,20 @@ The Operator SDK is a framework that uses the [controller-runtime][controller_ru
 
 The SDK provides workflows to develop operators in Go or Ansible.
 
-The following workflow is for a new Go operator:
+The following workflow is for a new **Go** operator:
 1. Create a new operator project using the SDK Command Line Interface(CLI)
 2. Define new resource APIs by adding Custom Resource Definitions(CRD)
 3. Define Controllers to watch and reconcile resources
 4. Write the reconciling logic for your Controller using the SDK and controller-runtime APIs
 5. Use the SDK CLI to build and generate the operator deployment manifests
 
+The following workflow is for a new **Ansible** operator:
+1. Create a new operator project using the SDK Command Line Interface(CLI).
+2. Define new resource APIs by adding Custom Resource Definitions(CRD) (This can be done in the new command)
+3. Write the reconciling logic for your object using ansible playbooks and roles.
+4. Define the mapping between the CRD definition in the `watches.yaml` file.
+5. Use the SDK CLI to build and generate the operator deployment manifests
```

Comment:
Nit: Either keep or remove the period(full stop) at the end of the sentence for all bullet points.
4 has it. 5 doesn't.